feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -499,19 +500,20 @@ func TestSoftwareVersion(t *testing.T) {
 		if v, ok := parent.Val(); ok {
 			got := gnmi.Get(t, dut, gnmi.OC().Component(v).Type().State())
 
-			// Arista_7280 OC component EOS has parent type Chassis
-			if dut.Model() == "DCS-7280CR3K-32D4" {
-				if got == chassisType {
-					t.Logf("Got a valid parent %v with a type %v for the component %v", v, got, os)
-				} else {
-					t.Errorf("Got a parent %v with a type %v for the component %v, want %v", v, got, os, chassisType)
-				}
+			want := []oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT{supervisorType}
+			if deviations.OSComponentParentIsChassis(dut) {
+				want = []oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT{chassisType}
+			}
+			if deviations.OSComponentParentIsLinecard(dut) {
+				want = []oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT{supervisorType, linecardType}
```

Comment:
I don't get it. If the `deviations.OSComponentParentIsLinecard` is true, why are we permitting the parent to be of a supervisor type?

Reply:
For CISCO 8808 some OS components have linecard as parent while others have supervisor. I updated the deviation name to reflect that fact.